tutorials/light_ecModel/protocol.m

```diff
@@ -0,0 +1,138 @@
+% trailing version identifier, and there is no field in Uniprot that
+% exactly matches that format, a custom uniprotConversion.tsv is generated.
+% Based on the genes.tsv that is distributed with Human-GEM:
+% https://github.com/SysBioChalmers/Human-GEM/blob/194ebe5431c83e25f78df61caacad2fa485b5cb4/model/genes.tsv
+% of which the first and the fourth column are kept to make data/uniprotConversion.tsv.
+
+% The second parameter (true) here signifies that a light ecModel will be
+% generated
+[ecModel, noUniprot] = makeEcModel(model,true);
+
+% noUniprot contains 6 genes that could not be mapped, so the genes.tsv
+% from Human-GEM was not sufficient and additional curation should be done.
+% This will be skipped for now, as 6 genes is not too many.
+
+% STEP 4
+[ecModel, foundComplex, proposedComplex] = applyComplexData(ecModel);
+
+% STEP 6
+ecModel         = getECfromGEM(ecModel);
+kcatList_fuzzy  = fuzzyKcatMatching(ecModel);
+
+% STEP 7
+ecModel         = findMetSmiles(ecModel);
+% DLKcat.tsv files are ecModel-type specific and cannot be interchanged
+% between full and light ecModels. This is due to a difference in how
+% reactions are represented in the ecModel.ec structure (isoenzymic
+% reactions are split in full ecModels, and a suffix (e.g. _EXP_1) is
+% attached to reaction identifiers; while in light models such reactions
+% are not split and a counting prefix (e.g. 001_) is attached to reaction
+% identifiers.
+writeDLKcatInput(ecModel);
+runDLKcat();
+kcatList_DLKcat = readDLKcatOutput(ecModel);
+
+% STEP 8
+kcatList_merged = mergeDLKcatAndFuzzyKcats(kcatList_DLKcat, kcatList_fuzzy);
+
+% STEP 9
+ecModel = selectKcatValue(ecModel,kcatList_merged);
+
+% STEP 10
+% applyCustomKcats can be run on light ecModels in a similar way as for
+% full ecModels. However, no custom kcat values are defined for the ecModel
+% in this tutorial.
+
+% STEP 11
+% getKcatAcrossIsoenzymes cannot be run on light ecModels
+
+% STEP 12
+% getStandardKcat can be run on light ecModels in a similar way as for full
+% ecModels. But due to the different model structure, no "standard"
+% pseudo-enzyme is included 
+% 
+% Instead of introducing a "standard" pseudo-enzyme, the new
+% standard enzyme usage constraints follow the light ecModel structure,
+% where the prot_pool is included as pseudo-substrate with
+% -standardMW/standardKcat as the stoiciometric coefficient.
+[ecModel, rxnsMissingGPR, standardMW, standardKcat] = getStandardKcat(ecModel);
+
+% STEP 13
+% Also in light ecModels are the kcat values kept in ecModels.ec.kcat, and
+% only after running applyKcatConstraints are these introduced in the
+% ecModel.S matrix. In constrast to full ecModels, the applyKcatConstraints
+% function checks across isoenzymes which is the most efficient enzyme
+% (lowest MW/kcat value), and this value is used to introduce
+ecModel = applyKcatConstraints(ecModel);
+
+% STEP 14
+ecModel = setProtPoolSize(ecModel);
+sol=solveLP(ecModel,1)
```

Comment:
`'The problem is infeasible` 🤔 I'm running the protocol agian, hoping it was just a step I missed.

Reply:
I've tried this a few more times, same result. Perhaps a different version of the model was used? Or with different params @edkerk ?

Reply:
It was just the solver, somehow I ended up with `glpk`. Setting that to `gurobi` resolved it. I will propose a suggestion higher up in the protocol.